File: algorithms/a01/FizzBuzz.js
function fizzBuzz(n) {
        switch(true) {
                case(n % 3 === 0 && n % 5 === 0):
                    console.log("FizzBuzz");
                    break;
                case(n % 5 === 0):
                    console.log("Buzz");
                    break;
                case(n % 3 === 0):
                    console.log("Fizz");
                    break;
                default:
                    console.log(n);
    }
}
fizzBuzz(3)
fizzBuzz(5)
fizzBuzz(45)

File: algorithms/a01/FizzBuzz.java
import java.util.Scanner;

class FizzBuzz {
	public static void main(String[] args) {
    try (Scanner input = new Scanner(System.in)) {
      System.out.print("Input : ");
      int n = input.nextInt();
      for (int i = 0; i <= n; i++) {
        System.out.println(i % 15 != 0 ? i % 5 != 0 ? i % 3 != 0 ? i : "Fizz" : "Buzz" : "FizzBuzz");
      }
    }
  }
}
